cmd/sqlflowserver/main_test.go

```diff
@@ -1546,94 +1546,15 @@ func CaseTrainXGBoostOnPAI(t *testing.T) {
 
 func CaseTrainDenseCol(t *testing.T) {
 	a := assert.New(t)
-
-	// Test train and predict using preprocessed table contains origin raw category strings
-	trainSQL := `select label,COALESCE(NULLIF(l1, ''),0) AS ll1,COALESCE(NULLIF(l2, ''),0) AS ll2,COALESCE(NULLIF(l3, ''),0) AS ll3,COALESCE(NULLIF(l4, ''),0) AS ll4,COALESCE(NULLIF(l5, ''),0) AS ll5,COALESCE(NULLIF(l6, ''),0) AS ll6,COALESCE(NULLIF(l7, ''),0) AS ll7,COALESCE(NULLIF(l8, ''),0) AS ll8,COALESCE(NULLIF(l9, ''),0) AS ll9,COALESCE(NULLIF(l10, ''),0) AS ll10,COALESCE(NULLIF(l11, ''),0) AS ll11,COALESCE(NULLIF(l12, ''),0) AS ll12,COALESCE(NULLIF(l13, ''),0) AS ll13,C1,C2,C3,C4,C5,C6,C7,C8,C9,C10,C11,C12,C13,C14,C15,C16,C17,C18,C19,C20,C21,C22,C23,C24,C25,C26 from alifin_jtest_dev.sqlflow_ctr_train_raw
```

Comment:
Do we want to re-add this case to unit tests?

Reply:
Maybe not, this test is only added for testing CSV input format.